bot/ordersActions.js

```diff
@@ -148,8 +154,9 @@ const buildDescription = (
       tasaText =
         i18n.t('rate') + `: ${process.env.FIAT_RATE_NAME} ${priceMarginText}\n`;
     } else {
-      const exchangePrice = getBtcExchangePrice(fiatAmount[0], amount);
-      tasaText = i18n.t('price') + `: ${exchangePrice.toFixed(2)}\n`;
+      const regex = /\,|\./g;
+      const exchangePrice = getBtcExchangePrice(fiatAmount.replace(regex,''), amount.replace(regex,''));
```

Comment:
you are now manually unformatting the fiatAmount, it was unnecessarily formatted